hvpy/tests/test_facade.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+import pytest
+
+from hvpy import getJP2Image
+
+
```

Comment:
This is just a future comment:

We will want to consider parameterizing the tests to avoid having to repeat the same tests for very similar endpoints.

Reply:
What do you mean by this?

I agree that these tests are pretty much identical to the InputParameters tests. I think since `getJP2Image` is pretty much just a passthrough to the input parameters, we can use the facade for testing the input parameters instead of trying to test the facade itself.

Meaning, get rid of `test_facade.py` and and use this `getJP2Image` function in `test_get_jp2_image.py` to cover both facade and input params.

Reply:
What I was thinking was that 1) we dont need so many tests for `facade` since we have that covered for the proper tests under the api_groups and we will get to a point where the difference in some of the `facade` tests will just be the function names. 

So instead of defining a test per endpoint, you would parameterize the function called.